.github/workflows/run-all-e2e-test.yaml

```diff
@@ -0,0 +1,32 @@
+name: All e2e tests
+
+on:
+  pull_request:
+    branches: [ main ]
+    types: [ labeled ]
+
+jobs:
+  e2e-full:
+    if: ${{ github.event.label.name == 'pr/run-all-e2e-test' }}
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17
+
+    - name: Install ginkgo
+      run: go get github.com/onsi/ginkgo/ginkgo@v1.16.5
+
+    - name: turn off swap
+      run: sudo swapoff -a
+
+    - name: Set netfilter conntrack max
+      run: sudo sysctl -w net.netfilter.nf_conntrack_max=131072
+
+    - name: Run PR-Blocking e2e tests
+      run: yes | make test-e2e
```

Comment:
Just a note: We have a [clusterclass test ](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/main/.github/workflows/e2e.yml#L28) also, so we might need to add that in this flow.

Reply:
@shivi28 This workflow will run all the tests inside e2e directory. So we don't need to add another flow into that. The clusterclass test will be triggered using `make test-e2e` command.

Reply:
Great, then it's fine